### PR TITLE
sys-cluster/teleport: remove v2.4.2 v2.4.5 v2.5.6, add v2.5.8 v2.6.0

### PR DESCRIPTION
**Pull request by gclawes**
* remove v2.4.2 v2.4.5 v2.5.6
* add v2.5.8 v2.6.0 

Minor semver release v2.6.0: https://github.com/gravitational/teleport/releases/tag/v2.6.0

Teleport v2.5.8 contains important security update (https://github.com/gravitational/teleport/releases/tag/v2.5.8).  Remove all previous v2.5.* versions.

Teleport v2.4.* is deprecated

### CODE REVIEW

File: sys-cluster/teleport/teleport-2.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+fi
+
+IUSE=""
+LICENSE="Apache-2.0"
+RESTRICT="test strip"
+SLOT="0"
+
+DEPEND="
+	app-arch/zip
+	>=dev-lang/go-1.9.2"
```
Comment (Zlogene) [inline sys-cluster/teleport/teleport-2.6.0.ebuild]: Do not depend on go directly, wgere are you using zip?

File: sys-cluster/teleport/teleport-2.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils golang-build systemd user
```
Comment (Zlogene) [inline sys-cluster/teleport/teleport-2.6.0.ebuild]: where are you using eutils and user eclasses?

File: sys-cluster/teleport/teleport-2.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+src_install() {
+	keepdir /var/lib/${PN} /etc/${PN}
+	dobin src/${EGO_PN%/*}/build/{tsh,tctl,teleport}
+
+	insinto /etc/${PN}
+	newins "${FILESDIR}"/${PN}-2.6.yaml ${PN}.yaml
+
+	newinitd "${FILESDIR}"/${PN}.init.d ${PN}
```
Comment (Zlogene) [inline sys-cluster/teleport/teleport-2.6.0.ebuild]: what are strange extensions? `.conf` and `.init` are correct ones, to just sum up, all the existing ebuilds for the package are having the same mistakes, not only these ones

File: sys-cluster/teleport/teleport-2.6.0.ebuild (hunk 1 above)
Comment (gclawes) [reply sys-cluster/teleport/teleport-2.6.0.ebuild]: zip is used inside the Makefile for the package, to bundle the webassets in the main teleport binary: https://github.com/gravitational/teleport/blob/v2.6.0/Makefile#L156.

What's the recommended way to depend on go as a build dependency?

File: sys-cluster/teleport/teleport-2.6.0.ebuild (hunk 2 above)
Comment (gclawes) [reply sys-cluster/teleport/teleport-2.6.0.ebuild]: Remove eutils, user from 2.5.8, 2.6.0, 9999

File: sys-cluster/teleport/teleport-2.6.0.ebuild (hunk 3 above)
Comment (gclawes) [reply sys-cluster/teleport/teleport-2.6.0.ebuild]: `teleport.yaml` is the configuration file for teleport installed in `/etc/teleport`.

I there a reference for the correct extensions for these files?  The `teleport.init.d` file is being installed correctly under `/etc/init.d/teleport`

File: sys-cluster/teleport/teleport-2.6.0.ebuild (hunk 1 above)
Comment (Zlogene) [reply sys-cluster/teleport/teleport-2.6.0.ebuild]: go dependency provided by the eclass

Comment (gclawes) [reply sys-cluster/teleport/teleport-2.6.0.ebuild]: Fixed

File: sys-cluster/teleport/teleport-2.6.0.ebuild (hunk 3 above)
Comment (gclawes) [reply sys-cluster/teleport/teleport-2.6.0.ebuild]: Is this explanation sufficient?  Or is there documentation somewhere I can follow for these extensions?

**Pull request closed without merging**